### DISCUSSION TITLE
PrimeNg dropdown assigning visual default value to first item in the list

### DISCUSSION THREAD

**Discussion by andievester** [primefaces/community#429] [PrimeNG] [answered]
According to the documentation, the Angular Dropdown component should NOT assign a default value unless specified. My dropdowns are assigning the first item in my list of options as the default item, instead of displaying the placeholder. The "assignment" is smoke and mirrors as well, because when I log out the values of my form controls, they are null to begin with. Here is my code: 
```
<form [formGroup]="formGroup">
        <div class="selector">
            <span class="select-prompt">Please select a group</span>
            <p-dropdown formControlName="group" [options]="groups" optionLabel="name"
                placeholder="Select Group"></p-dropdown>
        </div>
    
        <div class="selector">
            <span class="select-prompt">Please select a context</span>
            <p-dropdown formControlName="context" [options]="context"
                placeholder="Select Context"></p-dropdown>
        </div>
    </form>

```
`context: Array<string> = ["Search", "Dashboard"];`

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

You should set `autoDisplayFirst` property to `false`. It's `true` by default and it displays the first option automatically.

**Reply by shh79** [reply]
this solution does not work for me, what should I do?

**Reply by StroeAdrianFF** [reply]
Same for me, not working

**Reply by josebaug98** [reply]
I think he means adding something like: [autoDisplayFirst] in the p dropdown tag

**Reply by DhaaraniCIT** [reply]
it is not working for me aswell `[autoDisplayFirst]="false"`

**Reply by Juliya2323** [reply]
try setting **key** in default option

**Comment by utamuratov** [comment]
I noticed an important thing: If you do not write placeholder this issue happens. So You have two ways: one of them is write `[autoDisplayFirst]="false"` second one is to write placeholder to your dropdown.
